plugins/pretokenise.js

```diff
@@ -236,12 +236,13 @@
       if (needSpaceBetween(lastCh, ch))
         resultCode += " ";
       if (ch>=LEX_OPERATOR_START) {
+        var len
```

Comment:
This was just a scoping issue, there were two definitions of `len` but they're in the same scope so the linter complains, if this method isn't preferred the inner ones could have different names I suppose?

Reply:
could we just change `var` to `let`?